src/BloomBrowserUI/bookEdit/css/editPaneGlobal.less

```diff
@@ -20,7 +20,7 @@ To handling the mis-match.*/
 
 /* Qtip won't work if these rules are inside the scoped DIV */
 .qtip-content {
-    font-family: @RegularFontStack; // for xmatter field definition bubbles
+    font-family: @BubbleFontStack; // for xmatter field definition bubbles
```

Comment:
FWIW: You can have bubbles on template pages as well.

What we eventually want is just UI font, and probably defined without variables, just... on Body I imagine?